coalib/bears/requirements/GoRequirement.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, package, version="", flag=""):
         :param package: A string with the name of the package to be installed.
         :param version: A version string. Leave empty to specify latest version.
         :param flag:    A string that specifies any additional flags, that
-                        would be used with ``flag``.
+                        are passed to the manager.
```

Comment:
this should be fixed in d7df94871f533e4b1bf30dd30cfc525dd559925a @Adrianzatreanu